### ISSUE TITLE
Bulk operation on tables with multiple dots in name throws exception.

### ISSUE THREAD

**Issue by hgGeorg** [zzzprojects/EntityFramework-Extensions#415]
### Description
Doing a bulk operation (testet with BulkSynchronize and BulkMerge) on a table containing multiple dot characters in its name, like `Entity.With.Multiple.Dots` will cause it to throw an Exception. EF Core and Sql Server do allow mutiple dot characters in their table names.

### Exception
```
(Inner) Exception message:
Invalid object name 'Entity.With'.

Stack trace:
   at Z.BulkOperations.BulkOperation.()
   at Z.BulkOperations.BulkOperation.Execute()
   at .BulkSynchronize[T](DbContext this, IEntityType entityType, IEnumerable`1 list, Action`1 options, Boolean forceSpecificTypeMapping)
   at .BulkSynchronize[T](DbContext this, IEnumerable`1 entities, Action`1 options)
   at BulkSyncTest.TableNameWithMultipleDots.Demo(MyDbContext context, MyDbContext syncContext) in C:\source\repos\BulkSyncTest\BulkSyncTest\TableNameWithMultipleDots.cs:line 17
```

### Fiddle or Project (Optional)
I've updated the demo project from my last issue: #406 
The lines
```
TableNameWithMultipleDots.Demo(context, syncContext);
Inheritance.Demo(context, syncContext);
```
demonstrate each of the issues.

https://github.com/hgGeorg/bulk-sync-test

### Further technical details
- EF version: 5.0.6
- EF Extensions version: 5.1.37
- Database Provider: Microsoft.EntityFrameworkCore.SqlServer

**Comment by JonathanMagnan** [comment]
Hello @hgGeorg ,

Thank you for reporting, we will look at it.

Best Regards,

Jon

**Comment by hgGeorg** [comment]
Hello @JonathanMagnan ,

do you have any updates on this issue? We'd like to have a bit more information or a rough estimate to plan our own development and business.

Best regards,
Georg

**Comment by hgGeorg** [comment]
Hello @JonathanMagnan ,

do you have any updates on this issue?

Best regards,
Georg

**Comment by JonathanMagnan** [comment]
Hello @hgGeorg ,

Sorry for the delay, we got some issues last time we tried due to how the library has been created (we didn't expect dot in table name). We will look to add an options this time as we cannot support this case unfortunately by default due to backward compatibility.

**Comment by JonathanMagnan** [comment]
Hello @hgGeorg ,

A new version has been released.

As said, we cannot do it by default but in this version, we added the option `DisableDotCheckForEscapeTableName= true`

It is possible to set it by default on your side:

```csharp
EntityFrameworkManager.BulkOperationBuilder = builder =>
{
	builder.DisableDotCheckForEscapeTableName= true;
};
```

Or by specific bulk operations.

Unfortunately as said, we cannot make a perfect solution without having an option, otherwise, we will always end up with some backward compatibility issue.

Let me know if that fix your issue

Best Regards,

Jon
